### DISCUSSION TITLE
How to test code coverage in development

### DISCUSSION THREAD

**Discussion by alhcomer** [WesternFriend/westernfriend.org#695] [General]
I am looking to contribute to the open issues and would like to contribute to the testing issues that are currently open.

How should I test my code coverage in development?

**Comment by brylie** [comment]
To run the project tests use the following command:

```sh
python manage.py test
```

You can specify a test suite by passing it to the `test` command using module notation, similar to imports

```sh
python manage.py test path.to.module.tests
```

Here are some more details about testing in Django:
- https://docs.djangoproject.com/en/4.1/topics/testing/overview/

If you would like to work on any of the issues, please leave a comment in the issue and I will assign you.
